docker/build/installers/install_perf.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2020 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# Fail on first error.
+set -e
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+#install perf
+apt-get -y update && \
+    apt-get -y install \
+    linux-tools-common \
+perf 2>&1 | grep linux-tools | head -n 1 | xargs sudo apt install -y
```

Comment:
?

Reply:
install perf include two step ,first linux-tools-common.the second according to perf tips to isntall the first package.